demo/specs/quickstart/README.md

```diff
@@ -3,42 +3,47 @@
 kubectl get pod -A
 ```
 
-#### Show the current MIG configuration of the machine
+#### Show the yaml files for the first 3 example apps discussed in the [KubeCon presentation](https://sched.co/1R2oG)
 ```console
-nvidia-smi --query-gpu=index,name,uuid,mig.mode.current --format=csv
-nvidia-smi -L
+vim -O gpu-test1.yaml gpu-test2.yaml gpu-test3.yaml
 ```
 
-#### Deploy the 4 example apps discussed in the slides
+#### Deploy the 3 example apps above 
 ```console
-kubectl apply --filename=gpu-test{1,2,3,4}.yaml
+kubectl apply --filename=gpu-test{1,2,3}.yaml
```

Comment:
Interesting -- I though I had purposfully removed demo 4 from this READMe for exactly this reason. Maybe I only did it here, but not in our actual driver repo: https://github.com/kubernetes-sigs/dra-example-driver

Reply:
Ah -- I see now. This is the file down in the quickstart folder -- yeah, this was just copied from my demo script when I presented this.

Reply:
Makes sense to update this to only include examples that can be run on any GPU and separate out the MIG use cases to another folder.

Reply:
Agree, test-5/6 are MIG examples too. We didn't even describe them in README. Let me update the PR.

Reply:
Created a new foler `mig` and moved the MIG example gpu-test4/5/6 to it and updated README files accordingly.